api/src/test/java/org/pmiops/workbench/reporting/ReportingUploadServiceTest.java

```diff
@@ -45,10 +45,7 @@
 import org.mockito.Captor;
 import org.pmiops.workbench.api.BigQueryService;
 import org.pmiops.workbench.db.model.DbUser;
-import org.pmiops.workbench.model.BillingStatus;
-import org.pmiops.workbench.model.ReportingSnapshot;
-import org.pmiops.workbench.model.ReportingUser;
-import org.pmiops.workbench.model.ReportingWorkspace;
+import org.pmiops.workbench.model.*;
```

Comment:
nit: the aou convention is not to use `import *`

Reply:
That is what I get from gradlew spotlessApply. How can I set it up correctly locally?

Reply:
spotlessApply shouldn't be introducing a `*` import, are you sure it's not your IDE? I believe Intellij is configured to do this by default on save. Note: some details on setting up a save hook for `google-java-format` (what spotlessApply is conforming to) are here: https://docs.google.com/document/d/1DtESBapEzvuti7xODTFPHorwmLM7LybF-6D5lhbIkLU/edit#heading=h.vj6ydv4ysw0p